motoko/hello-angular-motoko/package.json

```
@@ -0,0 +1,89 @@
+{
+  "name": "hello-angular-motoko",
+  "version": "0.0.3",
+  "author": {
+    "name": "Shawn Dotey",
+    "email": "insight431@protonmail.com",
+    "url": "https://forum.dfinity.org/u/infinityio.tech"
+  },
+  "description": "An example of a Motoko project with an Angular 14 frontend",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "TypeScript",
+    "Angular",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "",
+    "prebuild": "",
+    "prestart": "",
+    "init:dfx": "npm run restart:clean:dfx && npm run refresh:motoko && dfx stop",
+    "start": "npm run restart:dfx && npm run refresh:motoko && npm run serve:both && npm run print:ng",
+    "test": "npm run restart:dfx && ng test",
+    "deploy:dfx": "npm run restart:dfx && npm run build:prod && dfx deploy --network ic --no-wallet",
```

```suggestion
    "deploy:dfx": "npm run restart:dfx && npm run build:prod && dfx deploy --network ic",
```
Since dfx 0.9(?) this flag is implied and not necessary anymore.